### PR TITLE
use rimraf

### PR DESCRIPTION
**Pull request by gingur**
This adds support for removing the dist folder on windows machines during `prebuild` process. Ran into when attempting to install as a git url instead of npm version, which runs the build process.

### CODE REVIEW

**Comment by nicolo-ribaudo** [comment]
Why is this needed?

File: package.json [hunk 1]
```diff
@@ -25,13 +25,14 @@
...
     "test": "mocha",
-    "prebuild": "rm -rf dist/*",
+    "prebuild": "rimraf dist",
     "build": "rollup -c",
     "prepare": "npm run build",
     "preversion": "npm run lint && npm test"
   },
   "dependencies": {
     "@babel/helper-module-imports": "^7.7.4",
+    "rimraf": "^2.6.3",
```
Comment (nicolo-ribaudo) [inline package.json]: This should go in `devDependencies`

Comment (gingur) [reply package.json]: oops!

**Comment by Andarist** [comment]
@gingur please update also package-lock.json (just install this using npm)

**Comment by gingur** [comment]
@Andarist `rimraf` was already in the lock from `del` which is brought in via `lint-staged`. This is why I used an older version, for a better dedupe. The end result was no updates to the lock after performing a  `npm i -D rimraf@2.6.3`

**Comment by Andarist** [comment]
Ok, thanks for the explanation!

**Pull request merged**